server/api/routes/users.js

```diff
@@ -0,0 +1,106 @@
+const userRouter = require('express').Router();
+const bodyParser = require('body-parser');
+const jwt = require('jsonwebtoken');
+const cors = require('cors');
+const bcrypt = require('bcrypt');
+const { User } = require('../../db/Models/User');
+
+userRouter.use(cors());
+userRouter.use(bodyParser.json());
+userRouter.use(bodyParser.urlencoded({ extended: true }));
+
+process.env.SECRET_KEY = 'secret';
+
+userRouter.get('/users', async (req, res) => {
+  const users = await User.findAll();
```

Comment:
just curious, do we need to fetch all users, or this is just a temporary code?

Reply:
We will need it for the friends list in friends component